poetry.lock

```diff
@@ -1303,838 +1303,111 @@ python-versions = "^3.9"
-]
-virtualenv = [
-    {file = "virtualenv-20.14.1-py2.py3-none-any.whl", hash = "sha256:e617f16e25b42eb4f6e74096b9c9e37713cf10bf30168fb4a739f3fa8f898a3a"},
-    {file = "virtualenv-20.14.1.tar.gz", hash = "sha256:ef589a79795589aada0c1c5b319486797c03b67ac3984c48c669c0e4f50df3a5"},
-]
-webencodings = [
-    {file = "webencodings-0.5.1-py2.py3-none-any.whl", hash = "sha256:a0af1213f3c2226497a97e2b3aa01a7e4bee4f403f95be16fc9acd2947514a78"},
-    {file = "webencodings-0.5.1.tar.gz", hash = "sha256:b36a1c245f2d304965eb4e0a82848379241dc04b865afcc4aab16748587e1923"},
-]
-whitenoise = [
-    {file = "whitenoise-6.1.0-py3-none-any.whl", hash = "sha256:2067fe9008a3cd7d0d75f75c9240b54f5f59996ca285cbeab18fc1e89949e30d"},
-    {file = "whitenoise-6.1.0.tar.gz", hash = "sha256:6ccc6d1ad6fb688a398ea65d97db47f018ea0be7da75e146e8f2e837a04ba590"},
-]
-xhtml2pdf = [
-    {file = "xhtml2pdf-0.2.7-py3-none-any.whl", hash = "sha256:1b6410bd437a2fce37a5cfb1f27fdb2443c632199ee6aed7a82b7fec625c94ba"},
-    {file = "xhtml2pdf-0.2.7.tar.gz", hash = "sha256:292342f0436f4df77c40d2973bde4d0cb052dff00d6c7102d5530d761965bd6d"},
-]
+arabic-reshaper = []
+asgiref = []
+asn1crypto = []
+atomicwrites = []
+attrs = []
+blessings = []
+bpython = []
+certifi = []
+cffi = []
+cfgv = []
+charset-normalizer = []
+click = []
+codacy-coverage = []
+colorama = []
+coverage = []
+cryptography = []
+cssmin = []
+cssselect2 = []
+curtsies = []
+cwcwidth = []
+distlib = []
+dj-database-url = []
+django = []
+django-classy-tags = []
+django-cms = []
+django-filer = []
+django-formtools = []
+django-inline-svg = []
+django-js-asset = []
+django-mptt = []
+django-object-actions = []
+django-polymorphic = []
+django-sekizai = []
+django-sendmail-backend = []
+django-treebeard = []
+djangocms-admin-style = []
+djangocms-attributes-field = []
+djangocms-link = []
+djangocms-picture = []
+djangocms-text-ckeditor = []
+easy-thumbnails = []
+factory-boy = []
+faker = []
+filelock = []
+flake8 = []
+future = []
+greenlet = []
+gunicorn = []
+html5lib = []
+identify = []
+idna = []
+iniconfig = []
+isort = []
+jedi = []
+lxml = []
+mccabe = []
+nodeenv = []
+oscrypto = []
+packaging = []
+parso = []
+pdfkit = []
+pillow = []
+platformdirs = []
+pluggy = []
+pre-commit = []
+psycopg2-binary = []
+py = []
+pycodestyle = []
+pycparser = []
+pyflakes = []
+pygments = []
+pyhanko = []
+pyhanko-certvalidator = []
+pyparsing = []
+pypdf3 = []
+pypng = []
+pyqrcode = []
+pytest = []
+pytest-cov = []
+pytest-django = []
+python-bidi = []
+python-dateutil = []
+pytz = []
+pytz-deprecation-shim = []
+pyxdg = []
+pyyaml = []
+qrcode = []
+reportlab = []
+requests = []
+sentry-sdk = []
+six = []
+sqlparse = []
+stripe = []
+svglib = []
+tinycss2 = []
+toml = []
+tomli = []
+tqdm = []
+typing-extensions = []
+tzdata = []
+tzlocal = []
+unidecode = []
+uritools = []
+urllib3 = []
+virtualenv = []
+webencodings = []
+whitenoise = []
+xhtml2pdf = []
```

Comment:
@strokirk Har du någon aning varför den vill ta bort så mycket saker här... och borde vi inte kunna plocka bort hela `[metadata.files]`-taggen om den vill göra såhär egentligen?

Sen är det ju iofs en .lock-fil, så den borde vi väl inte in och peta i manuellt ändå...